packages/angular/build/src/tools/esbuild/budget-stats.ts

```diff
@@ -70,7 +73,7 @@ export function generateBudgetStats(
 
     stats.assets.push({
       // Component styles use the input file
-      name: Object.keys(entry.inputs)[0],
+      name: file,
```

Comment:
This can stay as it previously was. `file` is now the virtual output file which doesn't exist in the source.